project/Properties.scala

```diff
@@ -0,0 +1,5 @@
+object Properties {
+  val concurrentBuilds = Option(System.getProperty("build.concurrentBuilds")).map(_.toInt).getOrElse(2)
```

Comment:
Specify `-Dbuild.concurrentBuilds=<int>` to `sbt` to control the number of builds `buildAll` will run at a given time.